pipelines/pr.yaml

```diff
@@ -38,4 +38,38 @@ jobs:
       buildUWPArm: false
       buildUWPDotNet: false
       UnityVersion: $(Unity2018Version)
+  
+- job: Unity2019Validation
+  timeoutInMinutes: 90
+  pool:
+    name: On-Prem Unity
+    demands:
+    - ${{ variables.Unity2019Version }}
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+    - VS2019 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      buildUWPX86: false
+      buildUWPArm: false
+      buildUWPDotNet: false
+      UnityVersion: $(Unity2019Version)
```

Comment:
@CDiaz-MS Do we want to get just the 2019 change in for now, if we're not able to get 2020 working quite yet?

Reply:
Yes we can do that, I'll remove the 2020 validation from this file for now.